### PR TITLE
 Formatting of stories below fold is incorrect - GEM MOLO (FED)

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@peterp please review this commit: 
https://github.com/praekelt/molo-gem/pull/103/commits/f14903f4c87b2114dd4af62a3c48887048e28ede

Solve: Commented out old _style.scss and _enhanced.scss styles for a uniform button _styles.scss style
 
Old
issue-2070

New
screensolve


NOTE: I added it by mistake, already created it's own issue here:
https://github.com/praekelt/molo-gem/issues/105

File: gem/static/css/_enhanced.article-preview.scss [hunk 1]
```diff
@@ -36,16 +36,42 @@
...
+  .article-preview  {
+    padding: 0;
+    margin: 0 0 40px 0;
+    width: 50%;
+  }
+}
 
-
+.no-flexbox  .article-list  {
+  box-sizing: border-box;
```
Comment (peterp) [inline gem/static/css/_enhanced.article-preview.scss]: I don't think the device will support box-sizing if it doesn't support flex-box.

File: gem/static/css/_enhanced.article-preview.scss [hunk 2]
```diff
@@ -36,16 +36,42 @@
...
+  overflow: hidden;
 
   .article-preview {
     padding: 0;
     margin: 0 0 40px 0;
     width: 50%;
+    margin-left: 4.5%;
+    width: 45%;
```
Comment (peterp) [inline gem/static/css/_enhanced.article-preview.scss]: on linee 68 you've already got a width.

File: gem/static/css/_enhanced.article-preview.scss [hunk 3]
```diff
@@ -36,16 +36,42 @@
...
+    float: left;
+    position: relative;
+    max-height: 355px;
```
Comment (peterp) [inline gem/static/css/_enhanced.article-preview.scss]: I don't think we'll be able to work around the height issue.

File: gem/templates/profiles/viewprofile.html [hunk 4]
```diff
@@ -73,7 +73,7 @@
                         {{ password_change_form.confirm_password }}
                         <span>{{ form.confirm_password.errors }}</span>
                     </div>
-                    <input type="submit" value='{% trans "Change" %}'>
+                    <input type="submit" value='{% trans "Baguhin ang password" %}'>
```
Comment (peterp) [inline gem/templates/profiles/viewprofile.html]: This should be in English.

File: gem/templates/base.html [hunk 5]
```diff
@@ -27,6 +27,7 @@
           <link rel="stylesheet" type="text/x-scss" href="{% static '' %}css/style{% get_site_static_prefix %}.scss">
           <link rel="stylesheet" type="text/x-scss" media="screen and (min-width: 320px)" href="{% static '' %}css/enhanced{% get_site_static_prefix %}.scss">
         {% endcompress %}
+        <script src="{% static '' %}js/modernizr.2.8.3.js"></script>
```
Comment (peterp) [inline gem/templates/base.html]: This can be removed.

File: gem/static/css/style.scss [hunk 6]
```diff
@@ -194,8 +203,9 @@ a, a:hover {
     }
 
     input[type="submit"] {
-      width: 80%;
+      width: auto;
```
Comment (peterp) [inline gem/static/css/style.scss]: This is implied and can be removed.

File: gem/static/css/style.scss [hunk 7]
```diff
@@ -55,16 +55,26 @@ input, select {
...
+
+  height: auto;
+  line-height: 1.3333;
+  padding: 8px 10px;
+  @media screen and (min-width:480px) {
+    padding: 10px 24px;
+  }
 }
 
+/*button, .button, .gem-polls a  {
```
Comment (peterp) [inline gem/static/css/style.scss]: Please remove this comment.

File: gem/templates/profiles/viewprofile.html (hunk 4 above)
Comment (Mitso) [inline gem/templates/profiles/viewprofile.html]: Great, this is now fixed.

File: gem/static/css/enhanced.scss [hunk 8]
```diff
@@ -65,15 +65,18 @@ input, select {
...
-  padding: 0 24px;
-  height: 48px !important;
-  line-height: 48px !important;
   font-family: 'TwCenMT';
   font-size: 18px;
   font-weight: bold;
   text-transform: uppercase;
 }
 
+/*button,.button, .gem-polls a {
```
Comment (peterp) [inline gem/static/css/enhanced.scss]: Please remove the commented out code.

**Comment by Mitso** [comment]
@peterp please review

File: gem/static/css/_enhanced.article-preview.scss [hunk 9]
```diff
@@ -36,16 +36,32 @@
...
+      clear:both;
+  }
 
   .article-preview {
     padding: 0;
-    margin: 0 0 40px 0;
     width: 50%;
+    float: left;
+    position: relative;
+    height: 100%;
```
Comment (peterp) [inline gem/static/css/_enhanced.article-preview.scss]: You can remove this.

File: gem/static/css/_enhanced.article-preview.scss [hunk 10]
```diff
@@ -36,16 +36,32 @@
...
+  }
+  p {
+    margin: 0;
+  }
+}
 
+.article-list  {
+  overflow: hidden;
 
+  .article-preview:nth-child(odd){
```
Comment (peterp) [inline gem/static/css/_enhanced.article-preview.scss]: you can probably just apply clear to each `.article-preview`

Comment (Mitso) [inline gem/static/css/_enhanced.article-preview.scss]: Hmmm clear doesn't seem to work on each .article-preview but only on the third odd .article-preview item.

**Comment by Mitso** [comment]
Current issue:
float_issue

**Comment by Mitso** [comment]
@peterp @saeedmb8  please review this solution. All checks passed, I implemented Codie's solution for flake8 error.

File: setup.cfg [hunk 11]
```diff
@@ -1,6 +1,6 @@
 [flake8]
 exclude = ve,docs,migrations
-ignore = F403
```
Comment (saeedmb8) [inline setup.cfg]: @Mitso lets leave this for now in here

**Comment by saeedmb8** [comment]
@Mitso left a small comment, otherwise the flake8 issue looks good to me 👍

**Pull request merged**